### DISCUSSION TITLE
Physical interactions with Monika explanation

### DISCUSSION THREAD

**Discussion by TooMuchInvigilation** [Monika-After-Story/MonikaModDev#9896] [general]
Hi. Is there some kind of in-universe explanation of how physical interactions with Monika (like hugging and kissing) work? She seems to be able to feel the player's touch somewhat, but I'm not sure about the exact mechanics and I don't think Monika mentions anything about it. I'm thinking about making my own version of an additional interactions submod, but I want it to be as close to "canon" as possible.

**Comment by Booplicate** [comment]
I think it's intentionally open-ended for the player imagination. Generally, she can't directly see/feel you, which is why you always have to ask for that kind of interactions first. I always saw it as you two trying to do something at the same time what makes it special.

**Reply by ashyashton** [reply]
Yep, this. Like a long distance relationship where you have those little touch sensor bracelets or lamps, or when you're in a call and go "3... 2... 1... Play!" to watch a movie at the same time, or when you're texting and hug your phone since the other person isn't there. I always imagined it like that; we're essentially in the longest-distance long distance relationship ever with Monika so I pretty much view all interactions through that lens.
